src/components/carbon-provider/carbon-provider.component.tsx

```diff
@@ -15,6 +15,7 @@ export interface CarbonProviderProps extends NewValidationContextProps {
   children: React.ReactNode;
   /** Theme which specifies styles to apply to all child components. Set to `sageTheme` by default. */
   theme?: Partial<ThemeObject>;
+  ariaHidden?: "true" | "false";
```

Comment:
comment: I don't think we need this as a configurable prop, we could probably just add the attribute as I don't think. If it does need to be a prop we should match the attribute case:

```

  "aria-hidden"?: "true" | "false";
  
  ...
  "aria-hidden": ariaHidden = "false",